db/rdbms/src/main/java/io/camunda/db/rdbms/read/domain/VariableDbQuery.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.db.rdbms.read.domain;
+
+import io.camunda.search.filter.FilterBuilders;
+import io.camunda.search.filter.VariableFilter;
+import io.camunda.search.page.SearchQueryPage;
+import io.camunda.search.query.SearchQueryBase;
+import io.camunda.search.sort.SortOptionBuilders;
+import io.camunda.search.sort.VariableSort;
+import io.camunda.util.ObjectBuilder;
+import java.util.Objects;
+import java.util.function.Function;
+
+public record VariableDbQuery(VariableDbFilter filter, VariableSort sort, SearchQueryPage page) {
```

Comment:
❓ Why do we need to wrap them once again in another `VariableDbQuery` object? Putting the `VariableDbFilter` aside, why can't the `VariableQuery` be used instead? I noticed that the same is done in other places as well, like with `ProcessInstanceQuery`, but I couldn't find a reason for it.

Reply:
Hm, I'm not really sure, I think we did this to have the default values there. When I get rid of the VariableDbFilter, I can try to do it also without this wrapper class. But then I maybe have to set the Defaults in some layer above.

Reply:
Removed it and we now work directly with the VariableQuery